charts/sftp-server/README.md

```diff
@@ -54,6 +54,7 @@ helm upgrade sftp --install sj14/sftp-server
 | service.nodePort | int | `0` | When using the NodePort type, you can specify a fixed port ([Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport)) |
 | service.port | int | `22` |  |
 | service.type | string | `"ClusterIP"` |  |
+| service.annotations | object | `{}` | Annotations to add to the service |
```

Comment:
The readme is generated from the `values.yaml` file. Please adjust the `values.yaml` accordingly.

Reply:
Thanks for the clarification, missed that. Should be correct now.